x-pack/plugins/monitoring/server/kibana_monitoring/lib/get_collector_types_combiner.js

```diff
@@ -81,7 +88,7 @@ export function getCollectorTypesCombiner(kbnServer, config, _sourceKibana = sou
     // return new payload with the combined data
     // adds usage data to stats data
     // strips usage out as a top-level type
-    const result = [ statsResult, settingsResult ];
+    const result = [statsResult, settingsResult];
```

Comment:
This whoe method now correlates to a static method of the BulkUploader class in master: https://github.com/tsullivan/kibana/blob/528cc0d/x-pack/plugins/monitoring/server/kibana_monitoring/bulk_uploader.js#L142